index.bs

```diff
@@ -1266,9 +1278,11 @@ The <dfn method for=StorageManager>getDirectory()</dfn> method steps are:
 
 
 <h2 class=no-num id="acks">Acknowledgments</h2>
+<span lang=nl>Randell Jesup</span>
+(<a href=https://www.mozilla.org/>Mozilla</a>, <a href=mailto:rjesup@mozilla.com>rjesup@mozilla.com</a>
```

Comment:
It looks like Acknowledgement sections generally only list names or GitHub IDs (not emails) and follow the same (mostly) standardized format. Could we match that?

https://github.com/search?q=org%3Awhatwg+%22id%3D%5C%22acks%5C%22%3EAcknowledgments%3C%22&type=code